.gitignore

```diff
@@ -11,5 +11,8 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+# Build binary
+bin*
+
```

Comment:
Why don't you add vendor files here and lets not include that code in the repo. You are already using go modules which will handle downloading dependencies (as you do in your Dockerfile).

Reply:
yes, agreed. I saw other commits from others that included the vendor dir so I followed that pattern. As you mention the go modules gets around that need. I will submit another pr that removes the vendor stuff. I have other outstanding pr's and it was difficult to remove vendor dir without causing conflicts.